tools/reviewer/web/src/app/dashboard/reviews-panel/diff/code-block/comments/comments.component.ts

```diff
@@ -89,11 +89,11 @@ export class CommentsComponent implements OnInit {
   }
 
   addComment(): void {
-    const comment: Comment = {
-      content: this.textareaControl.value,
-      createdBy: this.authService.user,
-      timestamp: Date.now()
-    };
+    const comment = new Comment();
```

Comment:
There's no `Common.newBuilder()` pattern? In Java you do:
```
const comment = Comment.newBuilder()
    .setContent(...)
    .setCreatedBy(...)
    .build();
```

Reply:
I think there's no ability like that :(
```typescript
setContent(value: string): void;
```

set functions return nothing.